core/templates/dev/head/filtersSpec.js

```diff
@@ -453,6 +454,31 @@ describe('Testing filters', function() {
       'Only First lettEr is  Affected');
   }));
 
+  it('should not shorten the length of text', inject(function($filter) {
+    expect($filter('getAbbreviatedText')('It will remain unchanged.', 50))
+      .toBe('It will remain unchanged.');
+    expect($filter('getAbbreviatedText')(
+      'Itisjustaverylongsinglewordfortesting', 
```

Comment:
Try a word that has > 50 chars.

Reply:
Oh wait, you have one below. Sorry, ignore this.

Reply:
I think its done in a test below that.